### ISSUE TITLE
Weights 

### ISSUE THREAD

**Issue by sharad1999** [deepak112/Keras-SRGAN#34]
Can you please provide the weights of the model

**Comment by BassantTolba1234** [comment]
Please all,
I need the code of implementation this part 
the part is 

{The SRResNet networks
were trained with a learning rate of 10−4 and 106 update
iterations. We employed the trained MSE-based SRResNet
network as initialization for the generator when training
the actual GAN to avoid undesired local optima.{
